website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## September 2024
 
+- **New:** dbt Explorer now has trust signal icons, available in Preview. Trust signals offer a quick, at-a-glance view of data health when browsing your dbt models in Explorer. These icons indicate whether a model is Healthy, Caution, Degraded, or Unknown. For accurate health data, ensure the resource is up-to-date and has had a recent job run. Refer to [Trust signals](/docs/collaborate/explore-projects#trust-signals-for-models) for more information.
```

Comment:
adding bold since i think you're referring to the UI text/label for these icons. if this is too much, alternatively, you can unbold but use regular casing (no upper case letters).

```suggestion
- **New:** dbt Explorer now includes trust signal icons, which is currently available as a [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). Trust signals offer a quick, at-a-glance view of data health when browsing your dbt models in Explorer. These icons indicate whether a model is **Healthy**, **Caution**, **Degraded**, or **Unknown**. For accurate health data, ensure the resource is up-to-date and has had a recent job run. Refer to [Trust signals](/docs/collaborate/explore-projects#trust-signals-for-models) for more information.
```